tests/language/expressions/numeric-separators.js

```
@@ -0,0 +1,112 @@
+  test("separator in negative expressions", () => {
+    expect(-1_000).toBe(-1000);
+    expect(-0xFF_FF).toBe(-65535);
+  });
+
+  test("separator with scientific notation and sign", () => {
+    expect(1e+1_0).toBe(10000000000);
+    expect(1e-1_0).toBe(1e-10);
+  });
+
+  test("invalid: trailing separator is a syntax error", () => {
+    expect(() => eval("1_000_")).toThrow();
```

Biome (2.4.10)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)